src/components/Modal/Modal.vue

```diff
@@ -268,6 +275,35 @@ export default {
 				}, this.clearViewDelay)
 			}
 		},
+
+		/**
+		 * Toggle the slideshow state
+		 */
+		togglePlayPause() {
+			this.playing = !this.playing
+			if (this.playing) {
+				this.handleSlideshow()
+			} else {
+				clearTimeout(this.slideshowTimeout)
+			}
+		},
+
+		/**
+		 * Reset the slideshow timer and keep going if it was on
+		 */
+		resetSlideshow() {
+			this.$el.querySelector('.progress-ring__circle').classList.remove('animated')
```

Comment:
can't use use the `:class="{}` magic? or doesn't that work with svgs?

Reply:
Right, this is much nicer, I can use the next tick to trigger a `this.playing = !this.playing`
Vue is too clever, if you invert both on a function it won't change anything since it automatically detect that the change is worthless ^^